pkg/k8s/service.go

```diff
@@ -111,7 +111,7 @@ func ParseService(svc *types.Service) (ServiceID, *Service) {
 		// would introduce more complexity in already too complex LB codebase,
 		// so for now (until we have refactored the LB code) keep NodePort
 		// frontends in Service.NodePorts.
-		if svc.Spec.Type == v1.ServiceTypeNodePort {
+		if svc.Spec.Type == v1.ServiceTypeNodePort || svc.Spec.Type == v1.ServiceTypeLoadBalancer {
```

Comment:
Is this enough? does the service type loadbalancer has `port.NodePort`?

Reply:
> Is this enough?

Yes, I've tested manually.

> does the service type loadbalancer has port.NodePort?

Based on this comment https://github.com/brb/kubernetes/blob/master/pkg/apis/core/types.go#L3075, it should have the `port.NodePort` field being set.

Reply:
Also https://kubernetes.io/docs/concepts/services-networking/service/ : ```The Type field is designed as nested functionality - each level adds to the previous. This is not strictly required on all cloud providers (e.g. Google Compute Engine does not need to allocate a NodePort to make LoadBalancer work, but AWS does) but the current API requires it.```